src/hooks/job-schedules/index.js

```diff
@@ -0,0 +1,3 @@
+import useJobSchedules from './use-job-schedules'
+
+export { useJobSchedules }
```

Comment:
```suggestion
export { default as useJobSchedules } from './use-job-schedules'
```

Reply:
Addressed in [f53059b](https://github.com/dhis2/scheduler-app/pull/495/commits/f53059ba406181d3fce93f733bc6ae4249d73553)